docs/development.md

```diff
@@ -96,3 +97,15 @@
     ```sh
     wrk -t2 -c2 -d30s -s post.lua http://<api-service-address>/api/v1/projects/<my-project>/analyze
     ```
+
+
+## Running in kubernetes
+
+    1. If deploying from a private registry, verify that Kubernetes has access to the [Docker Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-aks).
+
+    2. If using a Kubernetes secert to manage the registry authentication, make sure it is registered under 'presidio' namespace
+
+    3. Edit charts/presidio/values.yaml to:
```

Comment:
you can link to the values.yaml file ... which is nicer.

Reply:
done